src/main/kotlin/app/revanced/patches/youtube/layout/utils/navbarindexhook/fingerprints/NavBarBuilderFingerprint.kt

```diff
@@ -0,0 +1,26 @@
+package app.revanced.patches.youtube.layout.utils.navbarindexhook.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
```

Comment:
@johnconner122 Can you help me navigate this fingerprint in JADX, I cannot found it

Reply:
`class public final Laeuo;`

`.super Ljava/lang/Object;`

And method name: `onClick`

YouTube v18.23.35

Reply:
Thanks

Reply:
That's not it, I'll let you know in a while.

Reply:
I have checked and cannot found it. Can you share the defpackage too ?

Reply:
###### Class defpackage.aeuq (aeuq)
.class public abstract Laeuq;
.super Landroid/widget/HorizontalScrollView;
.source "PG"


method: `.method public m(Landroid/content/Context;)V`

Reply:
Thanks